cinema-soft/src/main/java/softserve/academy/cinemasoft/utils/Validator.java

```diff
@@ -6,13 +6,13 @@
 import java.util.regex.Pattern;
 
 @Component
-public class Validator  {
+public class Validator {
 
-    public boolean isValid(String time){
+    public boolean isValid(String time) {
         Pattern pattern;
         Matcher matcher;
-        String time12HOURS_PATTERN = "([0|1]?[0-9]|2[0-3]):[0-5][0-9]";
-        pattern = Pattern.compile(time12HOURS_PATTERN);
+        String time12HoursPattern = "([0|1]?[0-9]|2[0-3]):[0-5][0-9]";
```

Comment:
Extract the pattern string as a constant for the class.

Reply:
Fixed.